.circleci/config.yml

```diff
@@ -99,7 +99,6 @@ jobs:
       - run:
           name: Build - Configure
           command: |
-            activate-gcloud-account.sh
             mkdir -p /tmp/workspace/var
             mkdir -p /tmp/workspace/src
             echo "${CIRCLE_BUILD_NUM}" > /tmp/workspace/var/circle-build-num
```

Comment:
Probably this will also fail. I think `CIRCLE_BUILD_NUM` is considered a environmental variable and also not shared with forks.

Reply:
[Looks like it's working](https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/1480/workflows/05ebeeb9-6a40-45c2-81b7-8cbc253c44f5/jobs/8848/steps), I think the built in variables are always available. https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables.

Reply:
But running into another issue now. `CIRCLE_BRANCH` has the value `pull/1152`, so that gets passed to the composer file and it tries to load it as a branch, which fails. So we would need to pass in the forking repository, so that we can add it to the composer file, like here https://stackoverflow.com/questions/13498519/how-to-require-a-fork-with-composer.